README.md

```diff
@@ -70,4 +70,6 @@ Run `man 5 sway` for information on the configuration.
 ## Running
 
 Run `sway` from a TTY. Some display managers may work but are not supported by
-sway (gdm is known to work fairly well).
+sway (gdm is known to work fairly well). You will probably want to have xwayland
+installed, since most wayland applications (including dmenu and sway) will not
```

Comment:
dmenu is not a wayland application

Reply:
Thanks - I actually mentioned `dmenu` here because it is included in the default configuration. I'll rewrite it